libs/datasets/sources/can_scraper_local_dashboard_providers.py

```diff
@@ -126,6 +126,13 @@ def make_scraper_variables(provider: str):
             provider=provider,
             common_field=CommonFields.VACCINATIONS_COMPLETED,
         ),
+        ccd_helpers.ScraperVariable(
+            variable_name="total_vaccine_additional_dose",
+            measurement="cumulative",
+            unit="people",
+            provider=provider,
+            common_field=CommonFields.VACCINATIONS_ADDITIONAL_DOSE,
+        ),
```

Comment:
I assume this won't actually pick up any data but sets us up for scraping booster data in the future? 👍